.github/workflows/main.yml

```diff
@@ -17,12 +17,15 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 14.x
+          node-version: 16.x
           cache: 'yarn'
 
       - name: Install and build
         run: yarn install --force
 
+      - name: Install Docusaurus modules
+        run: cd ./docusaurus && yarn install --force
+
```

Comment:
Is this even needed?

Reply:
Feature wise - no. 

It's there because eslint throws errors due to to the fact that it cannot find imported modules in paths. E.g. `clsx`. 

So the options were at this moment:
1. Ignore whole `./docusaurus` folder and exclude it from linting checks altogether
2. Satisfy the linter. 

I went with the latter, since imo we should follow the chosen coding convention regardless on the submodule in the repo. 

If you have any other ideas how we could deal with it, I'm happy to look into it. Maybe we could add Docusaurus with Bob and migrate this integration to anothe sub-package, but it doesn't feel correct. After, it's not a part of the package itself and has nothing to do with the exported APIs. 

WDYT? @thymikee @mdjastrzebski

Reply:
So, what are we doing with this guys?

Reply:
Oh, that makes sense. Let's leave it then :)